### PR TITLE
Contribute to Kubernetes cloud explorer

### PR DESCRIPTION
**Pull request by JessicaJHee**
Closes https://github.com/redhat-developer/vscode-openshift-tools/pull/2620

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -795,6 +797,16 @@
 				"command": "openshift.component.deleteSourceFolder",
 				"title": "Delete Source Code Folder",
 				"category": "OpenShift"
+			},
+			{
+				"command": "openshift.sandbox.open.dashboard",
+				"title": "Open Console",
```
Comment (mohitsuman) [inline package.json:803]:
```suggestion
				"title": "Open Developer Console",
```

File: package.json [hunk 2]
```diff
@@ -795,6 +797,16 @@
...
+				"category": "OpenShift"
+			},
+			{
+				"command": "openshift.sandbox.signup",
+				"title": "Signup for Sandbox",
```
Comment (mohitsuman) [inline package.json:808]:
```suggestion
				"title": "Provision OpenShift Developer Sandbox",
```

**Review by mohitsuman — CHANGES_REQUESTED** [review]
wording changes.

**Review by msivasubramaniaan — APPROVED** [review]
LGTM

Tested in my local and it works as expected

**Comment by mohitsuman** [comment]
@JessicaJHee Let's add tooltip for Developer Sandbox and OpenShift Local.

**Comment by JessicaJHee** [comment]
> @JessicaJHee Let's add tooltip for Developer Sandbox and OpenShift Local.

Here's what it looks like now:
Peek 2023-06-19 14-04

File: src/cloudProvider/redhatCloudProvider.ts [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+            command: 'openshift.sandbox.open.setup',
+            title: 'Set up Developer Sandbox',
+        }
+        return sandboxItem;
+    }
+
+    private buildOpenshiftLocalItem() {
+        const openshiftLocalItem = new RedHatCloudItem('Openshift Local');
+        openshiftLocalItem.tooltip = 'Provision OpenShift 4 cluster to your local computer.';
+        openshiftLocalItem.iconPath = new vscode.ThemeIcon('symbol-folder');
```
Comment (mohitsuman) [inline src/cloudProvider/redhatCloudProvider.ts:87]: @JessicaJHee We should use the `vm` icon here for OpenShift local.

File: src/cloudProvider/redhatCloudProvider.ts [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+            tooltip: 'Create Red Hat Account'
+        }
+        return [loginItem, signUpItem];
+    }
+
+    private async buildDevSandboxItem() {
+        const signupStatus = await RedHatTreeDataProvier.getSandboxSignupStatus();
+        const sandboxItem = new RedHatCloudItem('Developer Sandbox');
+        sandboxItem.tooltip = 'Get 30-days free access to a shared OpenShift and Kubernetes cluster.';
+        sandboxItem.iconPath = new vscode.ThemeIcon('vm');
```
Comment (mohitsuman) [inline src/cloudProvider/redhatCloudProvider.ts:70]: @JessicaJHee We should use the `openshift logo` icon here instead of vm one.

**Review by mohitsuman — COMMENTED** [review]
Change the icons for the cluster providers.

**Review by mohitsuman — APPROVED** [review]
lgtm

**Pull request merged**